src/repository/DeviceKeyDatabaseRepository.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+import type { IBaseDatabase, IDeviceKeyRepository } from '../types';
+
+class DeviceKeyDatabaseRepository implements IDeviceKeyRepository {
```

Comment:
Because we have different `update` args..here update(deviceID, key), in other repos update(model). But actually, I think I'll make it consistent with others, just need to change the fileRepo a bit and places where we use the method.